integration_tests/test_sign_offline.py

```diff
@@ -1,66 +1,112 @@
 import pytest
-from chainlibpy import Transaction, Wallet
-from pystarport import ports
-from pystarport.proto_python.api_util import ApiUtil
+from chainlibpy import CROCoin, GrpcClient, NetworkConfig, Transaction, Wallet
+from chainlibpy.generated.cosmos.bank.v1beta1.tx_pb2 import MsgSend
 
 from .utils import wait_for_new_blocks
 
 pytestmark = pytest.mark.normal
 
 
-@pytest.mark.skip(reason="/txs API removed in 0.44")
+LOCAL_TESTNET = NetworkConfig(
+    # grpc_endpoint from data/chaintest/nodex/config/app.toml
+    # Look for "gRPC Configuration" section
+    grpc_endpoint="0.0.0.0:10003",
```

Comment:
in theory, `0.0.0.0` means all addresses on this host, but in practice connecting to `0.0.0.0` seems to be equivalent to `127.0.0.1`. But I guess still better to use `localhost` or `127.0.0.1` if this config is for the client to connect.